OpenRA.Mods.D2k/Traits/MakesNoise.cs

```diff
@@ -0,0 +1,36 @@
+#region Copyright & License Information
+/*
+* Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+* This file is part of OpenRA, which is free software. It is made
+* available to you under the terms of the GNU General Public License
+* as published by the Free Software Foundation. For more information,
+* see COPYING.
+*/
+#endregion
+
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.D2k.Traits
+{
+	[Desc("This actor makes noise.")]
```

Comment:
I think this should be clarified to state that this is an in-game (logic) system.

This could be misinterpreted as a way to play audio files.